docs/plugins/body_limit.md

```diff
@@ -0,0 +1,23 @@
+# Body Limit
+
+Block incoming requests whose body is greater than a specific size in megabytes.
+
+## Configuration
+
+The plain request transformer config:
+
+```json
+"body_limit": {
+    "enabled": true,
+    "config": {
+        "limit": "40M"
+    }
+}
+```
+
+Here is a simple definition of the available configurations.
+
+| Configuration                 | Description                                                         |
+|-------------------------------|---------------------------------------------------------------------|
+| name                          | Name of the plugin to use, in this case: body_limit        |
+| config.limit      | Allowed request payload size, default is `128M`. You can set the size in `B` for bytes,`K` for kilobytes, `M` for megabytes, `G` for gigabytes and `T` for terabytes  |
```

Comment:
I do not see where you set this default value. And `128M` seems too much - maybe `8M` or `16M` is enough?